### PR TITLE
118 pesel validation

### PR DESCRIPTION
**Pull request by damianwojtkowski**
Added PESEL validator in form based on #118 .
Implemented conditions to enable the user to click "DALEJ" button:
* length - 11 characters,
* valid check sum with last character of PESEL.

### CODE REVIEW

File: src/components/Dashboard/Form/FieldWrapper/FieldWrapper.js [hunk 1]
```diff
@@ -8,15 +8,19 @@ import PropTypes from "prop-types";
 import OptionList from "./SelectOptions";
 
 const required = value => (value ? undefined : "Required");
-
+const composeValidators = (required, validator) => value => {
+  if (typeof validator === "undefined") return required(value);
+  return validator(value);
+};
```
Comment (Maciek246) [inline src/components/Dashboard/Form/FieldWrapper/FieldWrapper.js]: Maybe can we move all validators + composeValidators into one file?

File: src/utils/DataHelper.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+  weights.forEach((weight, index) => {
+    sum += weight * pesel.substring(index, index + 1);
+  });
+
+  const checkSum = sum % 10 === 0 ? 0 : 10 - (sum % 10);
+
+  if (checkSum !== +pesel.substring(10, 11)) return "Invalid PESEL";
+
+  return undefined;
```
Comment (Maciek246) [inline src/utils/DataHelper.js]: Is there required? I guess if a function has no return at the end by default return `undefined`

File: src/components/Dashboard/Form/FieldWrapper/FieldWrapper.js (hunk 1 above)
Comment (damianwojtkowski) [reply src/components/Dashboard/Form/FieldWrapper/FieldWrapper.js]: I've moved validators and composeValidators into one file.

File: src/utils/DataHelper.js (hunk 2 above)
Comment (damianwojtkowski) [reply src/utils/DataHelper.js]: Yes, you are right. If function has no `return`, it returns `undefined` by default.

`return` has been removed from this function.

File: src/utils/validators.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+const required = value => (value ? undefined : "Required");
+
+export const isPeselValid = pesel => {
```
Comment (Maciek246) [inline src/utils/validators.js]: Please add a few unittests for checking if validators work are as expected.

Comment (Maciek246) [reply src/utils/validators.js]: Just create separated file e.g. `validators.test.js` and call a function with parameters.

Comment (damianwojtkowski) [reply src/utils/validators.js]: Test file with pesel validator test cases have been added.

**Comment by AgataSroka** [comment]
Conditions work correctly.
But comments should be written in Polish.

**Pull request merged**